aws-android-sdk-connect/src/main/java/com/amazonaws/services/connect/AmazonConnectClient.java

```diff
@@ -297,9 +363,9 @@ private void init() {
 
         HandlerChainFactory chainFactory = new HandlerChainFactory();
         requestHandler2s.addAll(chainFactory.newRequestHandlerChain(
-                "/com/amazonaws/services/Connect/request.handlers"));
```

Comment:
The change fixes the bug. It should be `connect` as opposed to `Connect`. This could be a mistake if the config.json used to generate code in the past may have had a typo with the character case.

Thank you for adding the config.json internally.